internal/testing/check_api_change.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Go Cloud Development Kit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script checks to see if there are any incompatible API changes on the
+# current branch relative to master@HEAD.
+# It fails if it finds any, unless there is a commit with BREAKING_CHANGE_OK
+# in the first line of the commit message.
+
+set -euo pipefail
+
+echo "Checking for incompatible API changes relative to master@HEAD..."
+echo
+
+go install -mod=readonly golang.org/x/exp/cmd/apidiff
+
+MASTER_CLONE_DIR="/tmp/go-cloud"
```

Comment:
Any reason this isn't another `$(mktemp)`?

Reply:
No. Done.